### PR TITLE
fix: Disabled using the erp command on self

### PR DESCRIPTION
**Pull request by Savage912-Official**
You can no longer use the erp command on yourself.

Issue: https://usarrp.gg/forums/topic/1410-do-not-erp-yourself/

### CODE REVIEW

**Comment by minipunch** [comment]
image

**Comment by Savage912-Official** [comment]
> image

You can use it on yourself and it glitches you lol

File: resources/[erp]/tgiann-skillbar/server/sv_cmds.lua [hunk 1]
```diff
@@ -88,8 +88,13 @@ TriggerEvent('es:addCommand', 'erp', function(source, args, char, location)
 		TriggerClientEvent("usa:notify", source, "Can't use that here")
 		return
 	end
-    local id = args[2]
-    TriggerClientEvent("erp:erp", source, id)
+   	local id = args[2]
+	local me = GetPlayerPed(source)
+	if not me then
```
Comment (minipunch) [inline resources/[erp]/tgiann-skillbar/server/sv_cmds.lua:93]:
```suggestion
	if source ~= id then
```

Comment (minipunch) [reply resources/[erp]/tgiann-skillbar/server/sv_cmds.lua:93]: Why not do this instead and that way you don't have to use `GetPlayerPed`

I wouldn't even think it works like you have it lol

**Review by minipunch — CHANGES_REQUESTED** [review]

File: resources/[erp]/tgiann-skillbar/server/sv_cmds.lua (hunk 1 above)
Comment (Savage912-Official) [reply resources/[erp]/tgiann-skillbar/server/sv_cmds.lua:93]: It worked but yeah I’ll change

**Review by minipunch — APPROVED** [review]

**Pull request merged**